uf3/regression/least_squares.py

```diff
@@ -1122,3 +1119,28 @@ def find_pair_potential_well(coefficients, rounding_factor):
             # no actual well
             well_idx = peak_idx + 1
     return well_idx
+
+
+def calculate_E_F_weights(n_e, n_f, e_stddev, f_stddev):
```

Comment:
The naming convention of this method is my only concern. Firstly the suffix `stddev` is too verbose. Please shorten this to `std`. Secondly, please be consistent in your naming `n_e` has the form `Property_variable` while `e_stddev` has the form `variable_property`. 

Lastly, the method name seems long. This is a personal preference but I would suggest the name `calc_E_F_weights`. I will leave that choice to you

Reply:
@JasonGibsonUfl Done